proto/panacea/oracle/v2alpha2/oracle.proto

```diff
@@ -123,4 +123,13 @@ enum VoteOption {
 
   // VOTE_OPTION_INVALID
   VOTE_OPTION_NO = 2;
+}
+
+// UpgradeOracleInfo defines the info of oracle upgrade, which includes the target height of upgrade and unique ID of the new version of oracle
+message OracleUpgradeInfo {
+  option (gogoproto.equal) = true;
```

Comment:
May I ask do you have any specific reason to set this option?

Reply:
This was simply copied from x/upgrade's proto and moved there.
This setting creates an equal function.

Since there is this setting in SoftwareUpgradeProposal, it is applied the same, and the Plan struct used internally also needs equal.

```go
func (this *Plan) Equal(that interface{}) bool {
	if that == nil {
		return this == nil
	}

	that1, ok := that.(*Plan)
	if !ok {
		that2, ok := that.(Plan)
		if ok {
			that1 = &that2
		} else {
			return false
		}
	}
	if that1 == nil {
		return this == nil
	} else if this == nil {
		return false
	}
	if this.UniqueId != that1.UniqueId {
		return false
	}
	if this.Height != that1.Height {
		return false
	}
	return true
}
```

Reply:
Oh the `Plan` struct used internally. Thanks for your explanation!